Apps/Sandcastle/gallery/development/Polylines.html

```diff
@@ -99,6 +99,24 @@
         material : Cesium.Material.fromType(Cesium.Material.PolylineArrowType)
     });
     Sandcastle.declare(localPolyline); // For highlighting on mouseover in Sandcastle.
+    
+    //Polyline using the fade material
+    var fadingPolyline = polylines.add({
+        positions : Cesium.Cartesian3.fromDegreesArrayHeights([-75, 43, 500000,
```

Comment:
Can you wrap this in `Cesium.PolylinePipeline.generateCartesianArc` so curves with the globe?  The other lines in this demo do that.